packages/react-native/ReactCommon/jsc/CMakeLists.txt

```diff
@@ -10,6 +10,15 @@
 cmake_minimum_required(VERSION 3.13)
 set(CMAKE_VERBOSE_MAKEFILE on)
 
+# Copy jsc headers into JavaScriptCore subdirectory
+get_target_property(JSC_PREFAB_INCLUDE_DIR jsc INTERFACE_INCLUDE_DIRECTORIES)
+set(JSC_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/include)
+file(MAKE_DIRECTORY ${JSC_INCLUDE_DIR})
+file(GLOB JSC_HEADERS "${JSC_PREFAB_INCLUDE_DIR}/*.h")
```

Comment:
IIRC we were potentially needing `CONFIGURE_DEPENDS` in places to invalidate build if underlying fetched files change? Nico would know more. Want to avoid any cases of missing or stale headers in copied folder.

Reply:
Does Maven package export headers without a namespace directory/as flat? I can see this is copied from `PrepareJSCTask`, but wonder if we have control over the directories exported to be able to just avoid the manipulation more generally.

Reply:
> `CONFIGURE_DEPENDS`

not sure how the case happens for the underlying prefab files change, but i can add `CONFIGURE_DEPENDS` anyway later

> Does Maven package export headers without a namespace directory/as flat?

it's totally controlled by maven package publisher. i just thought that for jsc-android. it doesn't make sense to add an extra nested directory for headers. e.g. `prefab/modules/jsc/include/JavaScript.h` -> `prefab/modules/jsc/include/JavaScriptCore/JavaScript.h`

it's just ios so we have to write [`#include <JavaScriptCore/JavaScript.h>`](https://github.com/facebook/react-native/blob/1d909efa235ffae150de25a201efbbe752d0bc52/packages/react-native/ReactCommon/jsc/JSCRuntime.cpp#L10) for the JavaScriptCore.framework

Reply:
Yup let's add CONFIGURE_DEPENDS here for the sake of consistency. We need it in the other targets as we do recursive globbing